auditbeat/module/file_integrity/scanner.go

```diff
@@ -55,10 +54,10 @@ func (s *scanner) Start(done <-chan struct{}) (<-chan Event, error) {
 				s.config.ScanRatePerSec,
 				s.config.MaxFileSize)
 
-		s.tokenBucket = ratelimit.NewBucketWithRate(
-			float64(s.config.ScanRateBytesPerSec)/2., // Fill Rate
-			int64(s.config.MaxFileSizeBytes))         // Max Capacity
-		s.tokenBucket.TakeAvailable(math.MaxInt64)
+		s.tokenBucket = rate.NewLimiter(
```

Comment:
Not sure we were using half the `ScanRateBytesPerSec` before? Everything else seems OK for the migration.

Reply:
That was a fudge factor. 🤦‍♂️